instrumentation/opentelemetry-instrumentation-annotations/opentelemetry-instrumentation-annotations-incubator/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/instrumentationannotations/incubator/counted/CountedExample.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.instrumentationannotations.incubator.counted;
+
+import io.opentelemetry.instrumentation.annotations.incubator.Attribute;
+import io.opentelemetry.instrumentation.annotations.incubator.Counted;
+import io.opentelemetry.instrumentation.annotations.incubator.ReturnValueAttribute;
+import io.opentelemetry.instrumentation.annotations.incubator.StaticAttribute;
+import java.util.concurrent.CompletableFuture;
+
+public class CountedExample {
+
+  public static final String METRIC_NAME = "name.count";
+  public static final String METRIC_DESCRIPTION = "I am the description.";
+  public static final String METRIC_UNIT = "ms";
+  public static final String TO_STRING = "I am a to string object.";
+
+  @Counted(METRIC_NAME)
+  public void exampleWithName() {}
+
+  @Counted(value = "example.with.description.count", description = METRIC_DESCRIPTION)
+  public void exampleWithDescription() {}
+
+  @Counted(value = "example.with.unit.count", unit = METRIC_UNIT)
+  public void exampleWithUnit() {}
+
+  @Counted("example.with.static.attributes.count")
+  @StaticAttribute(name = "key1", value = "value1")
+  @StaticAttribute(name = "key2", value = "value2")
+  @StaticAttribute(name = "key2", value = "value2")
```

Comment:
Line 32 and 33 are the same

Reply:
Sorry for late reply, I went to a family travel last week and come back just now. I think it's testing the key overlay case, so change it to:

```
  @Counted("example.with.static.attributes.count")
  @StaticAttribute(name = "key1", value = "value1")
  @StaticAttribute(name = "key2", value = "value3")
  @StaticAttribute(name = "key2", value = "value2")
```

and for TimedExample, change it to:

```
  @Timed(name = "example.with.static.attributes.duration")
  @StaticAttribute(name = "key1", value = "value1")
  @StaticAttribute(name = "key2", value = "value3")
  @StaticAttribute(name = "key2", value = "value2")
  public void exampleWithStaticAttributes() {}
```